.releaserc

```diff
@@ -17,15 +17,11 @@
     "@semantic-release/changelog",
     "@semantic-release/release-notes-generator",
     [
-      "@droidsolutions-oss/semantic-release-update-file",
+      "semantic-release-pub",
       {
-        "files": [
-          {
-            "path": ["pubspec.yaml"],
-            "type": "flutter",
-            "branches": ["main", "dev"]
-          }
-        ]
+        "cli": "flutter",
```

Comment:
I don't think this needs to be set. The publishPub field would prevent the use of the cli I would assume.